src/player.js

```diff
@@ -20,21 +20,22 @@ export class Player {
 		this.game = game;
 		this.creatures = [];
 		this.name = 'Player' + (id + 1);
+        let color = this.color
```

Comment:
This part is a bit odd. Wrong tabbing, uses spaces and also lacks semi colon at the end.

Reply:
fixed change using tabs for spacing and added semi colon at the end